lib/mongo/monitoring/event/secure.rb

```diff
@@ -40,6 +40,33 @@ module Secure
           'copydb'
         ].freeze
 
+        # Check whether the command is sensitive in terms of command monitoring
+        # spec. A command is detected as sensitive if it is in the
+        # list or if it is a hello/legacy hello command, and
+        # speculative authentication is enabled.
+        #
+        # @param [ String, Symbol ] command_name The command name.
+        # @param [ BSON::Document ] document The document.
+        #
+        # @return [ true | false ] Whether the command is sensitive.
+        #
+        # @since 2.1.0
```

Comment:
This @since isn't accurate and not needed.

Reply:
Removed